src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a
+          href="#"
+          class="logo"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="menu">
+            <li class="menu-item">
+              <a
+                class="menu-link is-active"
+                href="#"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="menu-item">
```

Comment:
Add the blank line between the links
```suggestion
            </li>
            
            <li class="menu-item">
```